### PR TITLE
Admin Catalog Menu and Sub Menu UI Cleanup

### PR DESCRIPTION
**Pull request by newlinewebdesign**
- Products Page
- Inventory Page
- Options Page
- Option Edit Page
- Vendors Page
- Vendor Edit Page
- Manufacturers Page
- Manufacturer Edit Page
- Filters Page
- Filter Group Edit Page
- Replaced jQuery inline code with Vanilla javascript included with web asset manager
- Added/Cleaned up additional language strings

### CODE REVIEW

File: administrator/components/com_j2store/controllers/manufacturers.php [hunk 1]
```diff
@@ -134,21 +143,38 @@ function add()
...
+			    }
+			    if (zoneElement) {
+			        zoneElement.classList.add('form-select');
+			        zoneElement.dispatchEvent(new Event('liszt:updated'));
+			    }
+			});
+			";
+	    $wa->addInlineScript($script, [], []);
+
+		/*echo '<script>
```
Comment (brianteeman) [inline administrator/components/com_j2store/controllers/manufacturers.php:170]: i would delete this block. No point in keeping commented out code

File: administrator/components/com_j2store/helpers/select.php [hunk 2]
```diff
@@ -131,28 +140,24 @@ public static function getOrderStatus($default_option = null, $asObject = false)
...
 
 	// get paymentlist
 	public static function PaymentList($selected = null, $id = '', $attribs = array(), $default_option = null) {
-		$paymentmethod_options [] = JHTML::_ ( 'select.option', '', JText::_ ( 'JALL' ) );
-
-		// ~ $paymentList = self::getPaymentList($default_option, true);
-		// ~ foreach($paymentList as $row) {
-		// ~ $paymentmethod_options[] = JHTML::_('select.option', $row->element, JText::_($row->element));
-		// ~ }
+		$paymentmethod_options [] = HTMLHelper::_( 'select.option', '', Text::_ ( 'JALL' ) );
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/select.php:160]:
```suggestion
		$paymentmethod_options [] = HTMLHelper::_( 'select.option', '', Text::_ ('JALL') );
```

File: administrator/components/com_j2store/layouts/list/default_filters.php [hunk 3]
```diff
@@ -1,54 +1,113 @@
...
+
+            <?php if(isset($field['type']) && $field['type'] === 'fieldsearchable'):?>
+                <?php $count++;?>
+                <?php $fieldsearchableCount++;?>
+                <?php if($count == 1) : ?>
+                    <div class="filter-search-bar btn-group flex-grow-1 flex-lg-grow-0 mb-2 mb-lg-0">
+                        <div class="input-group w-100 searchable_field">
+                            <input id="search_<?php echo $name;?>" type="text" name="<?php echo $name;?>" value="<?php echo $vars->state->get($name,'');?>" placeholder="<?php echo Text::_($field['label'])?>" class="form-control j2store-product-filters">
+                            <span class="filter-search-bar__label visually-hidden">
+                                <label id="search-lbl" for="search"><?php echo Text::_($field['label'])?></label>
```
Comment (brianteeman) [inline administrator/components/com_j2store/layouts/list/default_filters.php:45]: the id of the input is search_$name on line 43
```suggestion
                                <label id="search-lbl" for="search_<?php echo $name;?>"><?php echo Text::_($field['label'])?></label>
```

File: administrator/components/com_j2store/layouts/list/default_filters.php [hunk 4]
```diff
@@ -1,54 +1,113 @@
...
+                            } ?>
+
+                            <?php if(isset($field['type']) && $field['type'] === 'fieldsearchable'):?>
+                            <?php $count2++;?>
+                            <?php if($count2 > 1) : ?>
+                                <div class="col-lg-3 col-md-4 mb-2">
+                                    <div class="input-group w-100 searchable_field">
+                                        <input id="search_<?php echo $name;?>" type="text" name="<?php echo $name;?>" value="<?php echo $vars->state->get($name,'');?>" placeholder="<?php echo Text::_($field['label'])?>" class="form-control j2store-product-filters">
+                                        <span class="filter-search-bar__label visually-hidden">
+                                                <label id="search-lbl" for="search"><?php echo Text::_($field['label'])?></label>
```
Comment (brianteeman) [inline administrator/components/com_j2store/layouts/list/default_filters.php:99]:
```suggestion
                                                <label id="search-lbl" for="search_<?php echo $name;?>"><?php echo Text::_($field['label'])?></label>
```

File: language/backend/en-GB/en-GB.com_j2store.ini [hunk 5]
```diff
@@ -118,16 +118,20 @@ J2STORE_ADVANCED_SETTINGS="Advanced Settings"
 J2STORE_INTEGRITY_CHECK="Integrity Check"
 J2STORE_CONGRATULATIONS="Congratulations!"
 J2STORE_POSTCONFIG_WELCOME_MESSAGE="You are just a few steps away from building your online shop with J2Store"
-J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br> These can be managed at J2Store -> Configuration as well."
+J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br> These can be managed anytime at J2Store -> Configuration."
```
[the remark this answers is not in the document]
Comment (j2commerce) [reply language/backend/en-GB/en-GB.com_j2store.ini]: Thank you Brian!

**Pull request merged**